file_paths.md

```diff
@@ -237,84 +234,122 @@ create a `puppet` user or group.
+            extlookup2hiera.bat *
+            facter.bat *                        # bat file wrapper
             facter.exe *
+            facter_interactive.bat *
+            gem.bat *
+            hiera *                             # ruby bin wrapper
+            hiera.bat *                         # bat file wrapper
+            irb.bat *
+            libcpp-pcp-client.so *
+            libcurl-4.dll *                     # curl dll
+            libeay32.dll *                      # openssl dll
             libfacter.so *
-            lib*.dll *                        # gcc, mingw, boost libraries
-        inc *                                 # facter headers
-            facter *
-        lib *
-            facter.rb *                       # ruby bindings
-
-    C:\Program Files\Puppet Labs\Puppet\hiera *
-        bin *
-            hiera *                           # ruby bin wrapper
-        lib *
-            hiera.rb *
+            libgcc_s_seh-1.dll *
+            libstdc++-6.dll *
+            libwinpthread-1.dll *
+            mco *                               # ruby bin wrapper
+            mco.bat *                           # bat file wrapper
+            mco_daemon.bat *
+            mcollectived *                      # ruby bin wrapper
+            minitar *
+            minitar.bat *
+            nssm.exe *                          # NSSM used to run pxp-agent
+            openssl.exe *
+            puppet *                            # ruby bin wrapper
+            puppet.bat *                        # bat file wrapper
+            puppet_interactive.bat *
+            puppet_shell.bat *                  # targets for shortcuts
+            puppetres.dll *                     # event log message resource dll
+            pxp-agent.exe *
+            rake.bat *
+            rdoc.bat *
+            ri.bat *
+            ruby.exe *
+            rubyw.exe *
+            run_facter_interactive.bat *
+            run_puppet_interactive.bat *
+            ssleay32.dll *                      # openssl dll
+            stompcat *
+            stompcat.bat *
+            testrb.bat *
+            x64-msvcrt-ruby210.dll *            # ruby dll
+            zlib1.dll *
 
-    C:\Program Files\Puppet Labs\Puppet\mcollective *
-        bin *
-            mcollectived *                    # ruby bin wrapper
-            mco *
-        lib *
-            mcollective.rb *
+        include *
+            boost *
+            cpp-pcp-client *
+            curl *
+            facter *                            # facter headers
+            leatherman *
+            openssl *
+            ruby-2.1.0 *
 
-    C:\Program Files\Puppet Labs\Puppet\misc *
-        LICENSE.rtf *                         # license
-        puppetlabs.ico *                      # icon for start menu shortcut
-        puppetres.dll *                       # event log message resource dll
-        versions.txt *                        # versions of components
```

Comment:
Versions.txt needs to be included - should we continue with a misc directory for this purpose ?
Also - I think it is advisable to include the LICENSE.rtf file.